app.js

```diff
@@ -0,0 +1,6 @@
+import { start } from './src/index';
```

Comment:
When you are using ECMAScript modules you are should provide the file extension i.e. `index.js`